doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -98,6 +100,11 @@ To enable the FFS support, set the following configuration options to meet the A
 * :kconfig:option:`CONFIG_CHIP_DEVICE_TYPE` to the appropriate value, depending on the device used.
   The value must be compliant with the Matter Device Type Identifier.
 
+Every Matter device shall use an unique device identifier for rotating device identifier calculation purpose.
+In case of using factory data, the unique id is read from the factory data partion, otherwise the value set through the :kconfig:option:`CONFIG_CHIP_DEVICE_ROTATING_DEVICE_UID` is used.
```

Comment:
```suggestion
In case of using factory data, the unique identifier is read from the factory data partition.
In other cases, the value set through the :kconfig:option:`CONFIG_CHIP_DEVICE_ROTATING_DEVICE_UID` Kconfig option is used.
```